p2panda-rs/src/operation/operation_fields.rs

```diff
@@ -56,6 +60,7 @@ impl Validate for OperationValue {
         match self {
             Self::Relation(relation) => relation.validate(),
             Self::RelationList(relations) => relations.validate(),
+            Self::Owner(relation) => relation.validate(),
```

Comment:
I often daydream about using a `PinnedRelation` here..... It seems like quite an elegant solution to some of the complexities around members moving in and out of a key group.

Is that something we've seriously considered in recent times?